src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -3,9 +3,11 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Diagnostics;
 using System.Net;
 using System.Net.Http;
 using System.Net.Tests;
+using System.Reflection;
```

Comment:
Reflection does NOT work in UWP apps against CoreFx libraries.  It is basically turned off since CoreFx UWP libraries (including System.Net.Http) build process apply the __BlockReflectionAttribute.

So, these tests which use Reflection will not work when we start having our GitHub tests execute against the netcore50 (UWP) versions of the libraries. Right now the CI system only executes dnxcore50 versions of the built library binaries.

So, I'm wondering is the use of Reflection here just a specific test thing or does the architecture of this logging depend on Reflection in general.  Because if it does depend architecture-wise on Reflection, I don't understand how any of this Diagnostic logging will work in the UWP scenarios.